packages/mobile/src/networkInfo/reducer.ts

```diff
@@ -1,14 +1,21 @@
 import { REHYDRATE, RehydrateAction } from 'redux-persist'
 import { Actions, ActionTypes } from 'src/networkInfo/actions'
+import { UserLocationData } from 'src/networkInfo/saga'
 
 export interface State {
   connected: boolean // True if the phone thinks it has a data connection (cellular/Wi-Fi), false otherwise.
   rehydrated: boolean
+  userLocationData: UserLocationData
```

Comment:
Good call out. I'd rather add a migration so that we can assume `userLocationData` is defined everywhere it's used

Reply:
For simple cases like this, we can also avoid the migration with something like:
https://github.com/celo-org/wallet/blob/bd5c02e51b66daa37dfb4536fec23057c6daa225/packages/mobile/src/account/reducer.ts#L101

Reply:
Good to know. In your opinion, which is more appropriate here? I feel like a migration is more explicit and perhaps better?

Reply:
Additionally, how can we formulate a unit test around this? I think this is something that's easy to forget to do when introducing new elements to state

Reply:
If it's just for setting a default value I'm a fan of setting it in the rehydrate function because it's simpler and quicker. If it also needs migrating data or more complex calculations it's probably better to do a migration and unit test it.

Reply:
Not sure how to do this, but I guess something we could do is calling the reducer with an old state and the rehydrate action and check that the types of the fields of the state match the real thing.

I don't think there's an easy way to do that right now, but we could probably build something like that with some effort